src/devices/zigbeetlc.ts

```diff
@@ -0,0 +1,253 @@
+// requires zigbee2mqtt v1.34+
+// external converter for ZigbeeTLc v0.1.2.1+ by pvvx
+// https://github.com/pvvx/ZigbeeTLc
+// based on external converter for devbis-Firmware
+// https://raw.githubusercontent.com/devbis/z03mmc/master/converters/lywsd03mmc.js
+
+import {Zcl} from 'zigbee-herdsman';
+
+import * as types from '../lib/types';
+import * as constants from '../lib/constants';
+import * as reporting from '../lib/reporting';
+
+import {
+    battery,
+    binary,
+    enumLookup,
+    humidity,
+    numeric,
+    ota,
+    quirkAddEndpointCluster,
+    temperature,
+} from '../lib/modernExtend';
+
+const NS = 'zhc:zigbeetlc';
+
+async function configure(device: types.Zh.Device, coordinatorEndpoint: types.Zh.Endpoint, definition: types.Definition) {
```

Comment:
Can this one be removed now?

Reply:
This has now been removed as suggested. Device is working as expected.